pages/api/games/[id].js

```diff
@@ -0,0 +1,15 @@
+import dbConnect from "@/db/connect";
+import Games from "@/db/models/Game";
+
+export default async function handler(request, response) {
+  await dbConnect();
+  const { id } = request.query;
+  console.log("🚀  req:", request.query);
+
+  if (request.method === "GET") {
+    const games = await Games.findById(id);
```

Comment:
`game` over `games` - singular!

It would be also good to return a 404 if nothing was found, i.e.:

```js
if (!game) {
  return response.status(404).json({ message: 'Game not found' });
}
```